php/src/Communicator.cpp

```diff
@@ -516,7 +516,7 @@ ZEND_METHOD(Ice_Communicator, identityToString)
     CommunicatorInfoIPtr _this = Wrapper<CommunicatorInfoIPtr>::value(getThis());
     assert(_this);
 
-    zend_class_entry* identityClass = idToClass("::Ice::Identity");
```

Comment:
`idToClass` just called `nameToClass` after converting `::` to `\`.
The idea being that we were converting a type-id to a PHP class.

Now that there is no coupling between type-id and mapped named, may as well just pass the class name we know.